parachains/integration-tests/collectives/0_xcm/2_reserve.yml

```diff
@@ -42,43 +40,35 @@ tests:
                 - name: xcmPallet.Attempted
                   chain: *relay_chain
                   attributes:
-                    - type: XcmV2TraitsOutcome
+                    - type: XcmV3TraitsOutcome
                       xcmOutcome: Complete
-                      # the weight must be static
-                      value: *weight_to_send_reserve
                 - name: dmpQueue.ExecutedDownward
                   chain: *collectives_parachain
                   attributes:
-                    - type: XcmV2TraitsOutcome
+                    - type: XcmV3TraitsOutcome
                       xcmOutcome: Incomplete
-                      # the weight must be static
-                      value: [1000000000, UntrustedReserveLocation]
 
-      - name: Reserve assets from Collectives Parachain to Relay Chain fails
-        actions:
-          - extrinsics:
-            - chain: *collectives_parachain
-              signer: *alice_signer
-              pallet: polkadotXcm
-              call: reserveTransferAssets
-              args: [
-                { v1: { parents: 1, interior: { here: true }}}, # destination
-                { v1: { parents: 0, interior: { x1: { accountId32: { network: { any: true }, id: *alice_acc32 }}}}}, # beneficiary
-                { 
-                  v1: [ 
-                    { 
-                      id: { concrete: { parents: 1, interior: { here: true }}}, 
-                      fun: { fungible: 10000000000000 } # 10_000_000_000_000
-                    }
-                  ] 
-                }, # assets
-                0, # feeAssetItem
-              ]
-              events:
-                - name: system.ExtrinsicFailed
-                  attributes:
-                    - type: SpRuntimeDispatchError
-                      key: dispatchError
-                      # TODO assert variant
-                      # issue - https://github.com/paritytech/parachains-integration-tests/issues/59
-                      value: {"Module":{"index":"31","error":"0x02000000"}}
+      # Fails with the message - "RpcError: 1010: Invalid Transaction: Transaction would exhaust the block limits"
```

Comment:
@muharem
does this have any following issue or solution or it just stays commented out?

Reply:
we need parachains-integration-tests be able to handle it 
but better remove it for now

Reply:
removed